src/styles/blocks_main/_main-container.scss

```diff
@@ -0,0 +1,11 @@
+@import "./../variables.scss";
+
+.main_container {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: calc((4 * $card-width) + (3 * $card-spacing));
```

Comment:
you can rid of calc() when using `.scss`

Reply:
fixed